tests/admin_views/tests.py

```diff
@@ -6181,3 +6182,46 @@ def test_implicitly_generated_pk(self):
         post_data = {'name': '2'}
         response = self.client.post(reverse('admin:admin_views_implicitlygeneratedpk_change', args=(1,)), post_data)
         self.assertEqual(response.status_code, 302)
+
+
+@override_settings(ROOT_URLCONF='admin_views.urls')
+class AddM2MFieldsToObjChangeHistoryTest(TestCase):
```

Comment:
See if you can find an existing test case to use. There should be existing tests for log entries that this test could be located with.